mlflow/R/mlflow/tests/testthat.R

```diff
@@ -1,3 +1,21 @@
+# `trivial` is a fictional MLflow flavor that exists only for unit test purposes
+
+mlflow_save_model.trivial <- function(model, path, model_spec = list(), ...) {
+  if (dir.exists(path)) unlink(path, recursive = TRUE)
+  dir.create(path, recursive = TRUE)
+  path <- normalizePath(path)
+
+  trivial_conf = list(
+    trivial = list(type = "trivial")
```

Comment:
Are we sure this is exactly the spec we want? Also, looks like redundant information in here.... maybe I lack context.

Reply:
This one is just for the purpose of unit-testing. I think I can just replace this with something like `trivial = list(key1 = "value1", key2 = "value2")` as well :slightly_smiling_face: :